### PR TITLE
Demo site Branch for ONE37PM

### PR DESCRIPTION
**Pull request by veltsonbastien**
"ONE37pm sees the world through the lens of entrepreneurship—the ambition, the ​empathy, the ​grind. Own your future, start this minute." A media agency connected to Garyvee that I've been courting for a little while. Looking to keep the conversation going with a demo site.

### CODE REVIEW

File: src/pages/one37pm.tsx [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+          <mesh position={[2.49, 1.25, 4.2]} rotation-y={Math.PI / 2}>
+            <planeBufferGeometry args={[16.5, 3, 60, 30]} />
+            <meshStandardMaterial
+              color={0x46c1d1}
+              wireframe
+              transparent
+              opacity={0.8}
+            />
+          </mesh>
+        </group>
```
Comment (alex-shortt) [inline src/pages/one37pm.tsx:75]: you probably don't need this?

File: src/pages/one37pm.tsx [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+        socials={ARTIST.socialLinks}
+        name={ARTIST.name}
+        scenePos={[0, -20, 0]}
+        fogNear={0}
+        fogFar={150}
+        fogColor={"#000000"}
+        hMapScale={30}
+        xzMapScale={100}
+        floorColor="black"
+        night
```
Comment (alex-shortt) [inline src/pages/one37pm.tsx:54]: these are the wrong props for new gotham. make sure to run a `yarn build` to check for errors (or look into vs code/webstorm to set up eslint)

**Review by alex-shortt — CHANGES_REQUESTED** [review]

File: src/pages/one37pm.tsx (hunk 1 above)
Comment (veltsonbastien) [reply src/pages/one37pm.tsx:75]: I wanted them to have their colors on the mesh

File: src/pages/one37pm.tsx (hunk 2 above)
Comment (veltsonbastien) [reply src/pages/one37pm.tsx:54]: I will do so 👍

**Review by alex-shortt — CHANGES_REQUESTED** [review]
the latest link is not loading due to the images being inaccessible, i would recommend ripping them and hosting hem ourselves https://muse-place-gsxc2k8fq-metaplug.vercel.app/one37pm

**Review by alex-shortt — APPROVED** [review]

**Pull request merged**